build/test.mk

```diff
@@ -123,11 +123,17 @@ TEST_NAMES = \
 	TestIGCFilenameFormatter \
 	TestLXNToIGC \
 	TestLeastSquares \
-	TestThermalBand
+	TestThermalBand \
+	TestHexString
```

Comment:
To reduce the number of changed lines and improve "git blame", move the new line one up. Yes, this is an ugly problem.